### PR TITLE
Lint fixes

### PR DESCRIPTION
**Pull request by vrothberg**
Another set of fixes for golangci-lint reports. Now, all errors are fixed and we can migrate.

@rhatdan @TomSweeneyRedHat @nalind @saschagrunert PTAL

### CODE REVIEW

**Comment by saschagrunert** [comment]
@vrothberg Is this ready for review? I see you still pushin... :innocent:

**Comment by vrothberg** [comment]
> @vrothberg Is this ready for review? I see you still pushin... innocent

Ready for review. The repush was to fix a commit message. The CI isn't happy yet, so I still need to massage it a bit. I suspect that some errors should be ignored.

**Comment by TomSweeneyRedHat** [comment]
Should I close out #1666?

File: chroot/run.go [hunk 1]
```diff
@@ -1285,12 +1285,11 @@ func setupChrootBindMounts(spec *specs.Spec, bundlePath string) (undoBinds func(
 			}
 			isReadOnly := statfs.Flags&unix.MS_RDONLY != 0
 			// Check if any of the IDs we're mapping could read it.
-			isAccessible := true
+			isAccessible := false
```
Comment (TomSweeneyRedHat) [inline chroot/run.go]: can we move this down to have it above the next if statement at line 1293?

File: docker/types.go [hunk 2]
```diff
@@ -161,7 +161,7 @@ type V1Image struct {
 // V2Image stores the image configuration
 type V2Image struct {
 	V1Image
-	Parent     ID            `json:"parent,omitempty"`
+	Parent     ID            `json:"parent,omitempty"` //nolint
```
Comment (saschagrunert) [inline docker/types.go]:
```suggestion
	Parent     ID            `json:"parent,omitempty"` // nolint:govet
```

File: run_linux.go [hunk 3]
```diff
@@ -944,8 +944,8 @@ func setupRootlessNetwork(pid int) (teardown func(), err error) {
 	}
 
 	return func() {
-		cmd.Process.Kill()
-		cmd.Wait()
+		cmd.Process.Kill() // nolint
```
Comment (saschagrunert) [inline run_linux.go]:
```suggestion
		cmd.Process.Kill() // nolint:errcheck
```

File: run_linux.go [hunk 4]
```diff
@@ -944,8 +944,8 @@ func setupRootlessNetwork(pid int) (teardown func(), err error) {
...
+		cmd.Wait()         // nolint
```
Comment (saschagrunert) [inline run_linux.go]:
```suggestion
		cmd.Wait()         // nolint:errcheck
```

File: run_linux.go [hunk 5]
```diff
@@ -1131,11 +1131,11 @@ func runCopyStdio(stdio *sync.WaitGroup, copyPipes bool, stdioPipe [][]int, copy
 		if err := setNonblock(rfd, readDesc[rfd], true); err != nil {
 			return
 		}
-		setNonblock(wfd, writeDesc[wfd], false)
+		setNonblock(wfd, writeDesc[wfd], false) // nolint
```
Comment (saschagrunert) [inline run_linux.go]:
```suggestion
		setNonblock(wfd, writeDesc[wfd], false) // nolint:errcheck
```

File: run_linux.go [hunk 6]
```diff
@@ -1131,11 +1131,11 @@ func runCopyStdio(stdio *sync.WaitGroup, copyPipes bool, stdioPipe [][]int, copy
...
 	}
 
 	if copyPipes {
-		setNonblock(stdioPipe[unix.Stdin][1], writeDesc[stdioPipe[unix.Stdin][1]], true)
+		setNonblock(stdioPipe[unix.Stdin][1], writeDesc[stdioPipe[unix.Stdin][1]], true) // nolint
```
Comment (saschagrunert) [inline run_linux.go]:
```suggestion
		setNonblock(stdioPipe[unix.Stdin][1], writeDesc[stdioPipe[unix.Stdin][1]], true) // nolint:errcheck
```

File: tests/imgtype/imgtype.go [hunk 7]
```diff
@@ -89,7 +89,7 @@ func main() {
 
 	errors := false
 	defer func() {
-		store.Shutdown(false)
+		store.Shutdown(false) // nolint
```
Comment (saschagrunert) [inline tests/imgtype/imgtype.go]:
```suggestion
		store.Shutdown(false) // nolint:errcheck
```

File: cmd/buildah/rmi.go [hunk 8]
```diff
@@ -114,99 +114,98 @@ func deleteImages(ctx context.Context, systemContext *types.SystemContext, store
...
+		}
+		id, err := removeImage(ctx, systemContext, store, image)
+		if err != nil {
+			if lastError != nil {
+				fmt.Fprintln(os.Stderr, lastError)
+			}
+			lastError = errors.Wrapf(err, "error removing image %q", image.ID)
+			continue
+		}
+		fmt.Printf("%s\n", id)
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/rmi.go]: Just to make sure I've this change right, you just removed the `if image != nil {} here and moved everything over to the left one tab right?

Comment (vrothberg) [reply cmd/buildah/rmi.go]: Yep, that's it.

**Comment by vrothberg** [comment]
> Should I close out #1666?

I forgot about that entirely, sorry!

File: chroot/run.go (hunk 1 above)
Comment (vrothberg) [reply chroot/run.go]: Good catch! Done :heavy_check_mark:  (and soon to be pushed).

**Comment by vrothberg** [comment]
> [...] Generally, do you want to add a [config file](https://github.com/golangci/golangci-lint#config-file) to have a reproducible set of linters and options enabled/disabled?

Absolutely, that would be great. We can add that once we migrate the CI to golangci :+1:

**Comment by vrothberg** [comment]
> Generally LGTM, I just have some minor things point_down

Thanks a lot for checking :pray:  I fixed the corresponding commits and will push once I figured out why the CI is going south.

**Comment by vrothberg** [comment]
The regression is fixed and CI is looking to be green soon. @TomSweeneyRedHat @rhatdan @nalind PTA(final)L

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by vrothberg** [comment]
Awesome, so now we are unblocked on getting `golangci-lint` in :rocket:

**Pull request closed without merging**